core/services/relay/evm/multichain_transmitter_ocr3.go

```diff
@@ -0,0 +1,56 @@
+package evm
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+type MultichainMeta interface {
+	GetDestinationChainID() string
+}
+
+type multichainTransmitterOCR3[RI MultichainMeta] struct {
+	transmitters map[string]ocr3types.ContractTransmitter[RI]
+	lp           logpoller.LogPoller
+	lggr         logger.Logger
+}
+
+func NewMultichainTransmitterOCR3[RI MultichainMeta](
+	transmitters map[string]ocr3types.ContractTransmitter[RI],
+	lp logpoller.LogPoller,
+	lggr logger.Logger,
+) (*multichainTransmitterOCR3[RI], error) {
+	return &multichainTransmitterOCR3[RI]{
+		transmitters: transmitters,
+		lp:           lp,
+		lggr:         lggr,
+	}, nil
+}
+
+// FromAccount implements ocr3types.ContractTransmitter.
+func (m *multichainTransmitterOCR3[RI]) FromAccount() (types.Account, error) {
```

Comment:
Unclear what is the best thing to return here. If this value needs to be stored onchain, might be best that we hash the `strings.Join(accounts, ",")` at the bottom and return the address from that (lowest 160 bits)?